src/index.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-unused-vars */
+import _ from 'lodash';
+import './style.css';
+
+function component() {
+  const element = document.createElement('div');
+
+  return element;
+}
```

Comment:
**Changes Required**
- [ ] The use of `arrow function` is preferred to the `function` keyword. Let us refactor this into an ES6 arrow function. You can read more [here](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html) on why you should prefer ES6 arrow functions to the `function keyword`.